src/System.Security.Cryptography.OpenSsl/tests/EcDsaOpenSslTests.cs

```diff
@@ -294,5 +294,8 @@ internal static class Crypto
 
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcKeyDestroy")]
         internal static extern void EcKeyDestroy(IntPtr r);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_OpenSslVersionNumber")]
+        internal static extern uint OpenSslVersionNumber();
```

Comment:
But this function doesn't exist, right?  So probably don't want to define the P/Invoke for it

Reply:
Removed.